src/scss/lib/components/navbar.scss

```diff
@@ -0,0 +1,137 @@
+.navbar{
+  background-color: transparent;
+  padding: 22px 0 0;
+  .navbar-item{
+    color: #ffffff;
+    font-family: $font-menu,sans-serif;
+    font-size: 14px;
+    line-height: 18px;
+    background-color: transparent;
+    font-weight: 400;
+    align-self: center;
+    letter-spacing: 0.6px;
+    word-spacing: -2px;
+    margin-left: 41px;
+    &:hover{
+      background-color: transparent;
+      color: $gray-400;
+      transition: 0.3s ease-out;
+    }
+  }
+  .navbar-brand__logo{
+    position: relative;
+    left: 11px;
+    @include mobile{
+      margin-left: 5px;
+    }
+    img{
+      width: 31px;
+      height: 27px;
+    }
+  }
+  .navbar-menu{
+    position: relative;
+    right: 80px;
+    .button{
+      font-size: 11px;
+      font-weight: 400;
+      line-height: normal;
+      height: 32px;
+      width: 108px;
+      margin-left: 54px;
+      &:hover,&:active,&:focus{
+        background-color: transparent;
+        outline: none;
+        box-shadow: none;
+      }
+    }
+  }
+  .navbar-burger{
+    margin-right: 25px;
+    transition: 0.3s ease-out;
+    &.is-active{
+      span:nth-child(3) {
+        width: 34px;
+      }
+      span:nth-child(3) {
+        -webkit-transform: translateY(-5px) rotate(-45deg);
+        transform: translateY(-10px) rotate(-45deg);
+      }
+    }
+    span{
+      background-color: #ffffff;
+      width: 34px;
+      transition: 0.3s ease-out;
+      &:nth-child(2) {
+        top: calc(55% - 1px);
+      }
+      &:nth-child(3) {
+        width: 25px;
+        top: calc(60% + 4px);
+      }
+    }
+  }
+}
+
+.navbar-menu_mobile{
```

Comment:
dash and underscore at the same time, it's not consistent.

Reply:
fixed